doc/modules/clustering.rst

```diff
@@ -1185,7 +1179,18 @@ following equation, from Vinh, Epps, and Bailey, (2009). In this equation,
 Using the expected value, the adjusted mutual information can then be
 calculated using a similar form to that of the adjusted Rand index:
 
-.. math:: \text{AMI} = \frac{\text{MI} - E[\text{MI}]}{\max(H(U), H(V)) - E[\text{MI}]}
+.. math:: \text{AMI} = \frac{\text{MI} - E[\text{MI}]}{\text{mean}(H(U), H(V)) - E[\text{MI}]}
+
+For normalized mutual information and adjusted mutual information, the normalizing 
+value is typically some mean of the entropies of each clustering. Various means exist,
```

Comment:
Note that it is controlled by `average_method` in our implementatin

Reply:
I'm not sure if I like "some mean" because in probability theory there is only one mean. Maybe "some aggregate"? Also @jnothman's comment is not addressed, right?

Reply:
Right—It's some [generalized mean](https://en.wikipedia.org/wiki/Generalized_mean). The comment slipped through the cracks; I'll address it with the next update.

Reply:
Actually, it didn't slip through. It's addressed at the end of that paragraph.